fastlane/lib/fastlane/actions/spaceship_stats.rb

```diff
@@ -7,16 +7,31 @@ def self.run(params)
 
         rows = []
         Spaceship::StatsMiddleware.service_stats.each do |service, count|
-          rows << [service.name, service.url, service.auth_type, count]
+          rows << [service.name, service.auth_type, service.url, count]
         end
 
         puts("")
         puts(Terminal::Table.new(
                title: "Spaceship Stats",
-               headings: ["Service", "URL", "Auth Type", "Number of requests"],
+               headings: ["Service", "Auth Type", "Domain", "Number of requests"],
```

Comment:
Why "domain" but variable is called URL?

Reply:
I guess.. what shows _actually_ isn’t the domain but its a base url.... Changing it back!